presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -322,32 +325,47 @@ public void testRoundTrip(Type type, List<?> readValues)
         }
     }
 
-    private void testStructRoundTrip(Type type, List<?> readValues)
+    private void testStructRoundTrip(Type type, List<?> values)
```

Comment:
Why rename this variable? If it does need to be renamed (I don't think it should), then it should be done in a separate commit to keep the review simple.

Reply:
Ah I see.  That is ok then